cmd/otelcol/config/collector/gateway_config.yaml

```diff
@@ -27,6 +27,9 @@ receivers:
             regex: '.*grpc_io.*'
             action: drop
   sapm:
+  smartagent/signalfx-forwarder:
+    type: signalfx-forwarder
+    listenAddress: 0.0.0.0:9080
```

Comment:
Do we need this for gateway mode? We did not have this in signalfx gateway to my knowledge -- I think we should only enable this for agent mode.

Reply:
The smart gateway listened at this so I assumed we'd want the collector to as well.  If everything is reporting otlp then it's not necessary though.

Reply:
I think we should only do this for agent -- gateway mode config will be different and in gateway mode we should not enable signalfx-forwarder